.github/workflows/stale.yml

```diff
@@ -27,13 +27,13 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v4
+      - uses: actions/stale@v9.0.0
         with:
           # Stale Issues
           days-before-issue-stale: 30
           days-before-issue-close: 7
           # We do not stale Issues with label `Waiting for reply`, `feature` and `DSIP`
```

Comment:
LGTM. Comments need to be updated at the same time.

Reply:
Fixed. Thanks.